test/index.spec.js

```diff
@@ -754,3 +754,36 @@ describe('Batch', function () {
     });
   });
 });
+
+describe('Data Center config property name normalization', function () {
+  it('allows data_center', function () {
+    var api = new Cronofy({
+      data_center: 'de'
+    });
+
+    expect(api.urls.api).to.eq('https://api-de.cronofy.com');
+  });
+
+  it('allows dataCenter', function () {
+    var api = new Cronofy({
+      dataCenter: 'de'
+    });
+
+    expect(api.urls.api).to.eq('https://api-de.cronofy.com');
+  });
+
+  it('prefers data_center', function () {
```

Comment:
It might be worth logging a warning in this case- while it's probably rare, it's unlikely to be intentional.